src/prowjobsscraper/scraper.py

```diff
@@ -24,22 +26,35 @@ def execute(self, jobs: prowjob.ProwJobs):
         jobs.items = [j for j in jobs.items if self._is_assisted_job(j)]
 
         # filter out jobs already stored
-        known_build_ids = self._event_store.scan_build_ids()
-        jobs.items = [j for j in jobs.items if j.status.build_id not in known_build_ids]
+        known_jobs_build_ids = self._event_store.scan_build_ids_from_jobs_index()
+        jobs.items = [
+            j for j in jobs.items if j.status.build_id not in known_jobs_build_ids
+        ]
 
         # Retrieve equinix metadata for each job
-        self._equinix_extractor.hydrate(jobs)
+        self._equinix_metadata_extractor.hydrate(jobs)
 
         # Retrieve executed steps for each job
         steps = self._step_extractor.parse_prow_jobs(jobs)
 
+        # Retrieve equinix machines usages not already stored
+        known_usages_build_ids = self._event_store.scan_build_ids_from_usages_index()
+        usages = [
+            usage
+            for usage in self._equinix_usages_extractor.get_project_usages()
+            if usage.job_build_id not in known_usages_build_ids
+        ]
+
         # Store jobs and steps into their respective indices
         logger.info("%s jobs will be pushed to ES", len(jobs.items))
         self._event_store.index_prow_jobs(jobs.items)
 
         logger.info("%s steps will be pushed to ES", {len(steps)})
         self._event_store.index_job_steps(steps)
 
+        logger.info("%s equinix usages will be pushed to ES", {len(usages)})
```

Comment:
I guess the curly braces are not necessary, and I the line above too  :thinking: